.github/CODEOWNERS

```diff
@@ -11,9 +11,7 @@
 # - respond to issues
 # - fix failing unit tests or any other blockers to the CI/CD workflow
 # - update usage of `opentelemetry-python-core` APIs upon the introduction of breaking changes
-#
-# A CODEOWNER SHOULD:
-# - be a member of the OpenTelemetry community
+# - be a member of the OpenTelemetry community so that the `component-owners.yml` action to automatically assign CODEOWNERS to PRs works correctly.
```

Comment:
Our `component-owners.yml` action says this [in the its README](https://github.com/dyladan/component-owners#component-owners):

> This action only requires that component owners be collaborators on the repository or members of the organization, but does NOT require they have write access.

So we highlight that membership is required.